lib/vsc/install/commontest.py

```diff
@@ -69,6 +69,121 @@
 if sys.version_info >= (3, 5):
     HAS_PROSPECTOR = False
 
+# List of regexps patterns applied to code or message of a prospector.message.Message
+#   Blacklist: if match, skip message, do not check whitelist
+#   Whitelist: if match, fail test
+PROSPECTOR_BLACKLIST = [
+    # 'wrong-import-position',  # not sure about this, these usually have a good reason
+    'Locally disabling',  # shows up when you locally disable a warning, this is the point
+    'Useless suppression',  # shows up when you locally disable/suppress a warning, this is the point
+]
+# to dissable any of these warnings in a block, you can do things like add a comment # pylint: disable=C0321
+PROSPECTOR_WHITELIST = [
+    'undefined',
+    'no-value-for-parameter',
+    'dangerous-default-value',
+    'bare-except',
+    'E713',  # not 'c' in d: -> 'c' not in d:
+    'arguments-differ',
+    'unused-argument',
+    'unused-variable',
+    'reimported',
+    'F811',  # redefinition of unused name
+    'unused-import',
+    'syntax-error',
+    'E101',  # mixing tabs and spaces
+    'bad-indentation',
+    'E111',  # pep8: E111 / indentation is not a multiple of four
+    'bad-whitespace',
+    'trailing-whitespace',
+    'W291',  # pep8: W291 / trailing whitespace
+    #'protected-access',
+    #'logging-not-lazy',
+    'duplicate-key',  # when a key appears twice in a dict definition
+    # prospector cannot check duplicates across files, and pylint only detects duplicates across files (19/03/2019)
+    'duplicate-code', # when 4 or more lines of code show up several times
```

Comment:
trainling comments have 2 spaces? there's probbly a check for that somewhere (but i;m not sure we should enable it)

Reply:
we do not test stuff in the test directory. (its in the prospector ignore path)